packages/roosterjs-content-model-plugins/lib/autoFormat/AutoFormatPlugin.ts

```diff
@@ -113,8 +113,11 @@ export class AutoFormatPlugin implements EditorPlugin {
         if (rawEvent.inputType === 'insertText') {
             switch (rawEvent.data) {
                 case ' ':
-                    const { autoBullet, autoNumbering } = this.options;
+                    const { autoBullet, autoNumbering, autoLink } = this.options;
```

Comment:
let make these features off by default for now.
and consider add options to turn them on in demo site